source/jobs/sync-jobs.txt

```diff
@@ -54,6 +54,18 @@ To enable idempotent inserts for documents and embedded arrays, set
 ``user.properties`` file. Enabling idempotency may have substantial 
 performance impact on large jobs.
 
+Indexes
+~~~~~~~
+
+Relational Migrator creates new indexes in your MongoDB collection to
```

Comment:
(questions)
Does this happen automatically, with every job? 
Is it useful to state when in the sync job process this happens?

Reply:
@tomhollander  could you clarify here?

Reply:
Correct, RM will automatically create some basic additional indexes (beyond the built-in _id index) when relevant to a project mapping, eg when we have arrays and need to update the array contents.

Reply:
@davidsoulmdb Do you think it is useful to state when in the sync job process this happens?

Reply:
Discussed over slack and determined that this information is not useful for users to know.